cmake/configs/nuttx_px4fmu-v2_default.cmake

```diff
@@ -47,8 +47,8 @@ set(config_module_list
 	drivers/px4io
 	drivers/rgbled
 	drivers/sdp3x_airspeed
-	drivers/sf0x
-	drivers/sf1xx
+	#drivers/sf0x
```

Comment:
Please uncomment the `sf` drivers, as they are more widely supported and used by the community.

Reply:
Fixed

Output of `tfmini test`

'WARN  [tfmini] single read
WARN  [tfmini] measurement:  2.07 m
WARN  [tfmini] time: 114917001
WARN  [tfmini] read #0
WARN  [tfmini] valid 1
WARN  [tfmini] measurement:  2.070 m
WARN  [tfmini] time: 114928002
WARN  [tfmini] read #1
WARN  [tfmini] valid 1
WARN  [tfmini] measurement:  2.070 m
WARN  [tfmini] time: 114939004
WARN  [tfmini] read #2
WARN  [tfmini] valid 1
WARN  [tfmini] measurement:  2.070 m
WARN  [tfmini] time: 115441435
WARN  [tfmini] read #3
WARN  [tfmini] valid 1
WARN  [tfmini] measurement:  2.070 m
WARN  [tfmini] time: 115943013
WARN  [tfmini] read #4
WARN  [tfmini] valid 1
WARN  [tfmini] measurement:  2.070 m
WARN  [tfmini] time: 116445392
ERROR [tfmini] PASS`